### ISSUE TITLE
Teaching Guides #4

### ISSUE THREAD

**Issue by Hasmig** [GenEdProject/TeachGenocideSite#99]
For “Any Armenian Journey” there should also be an option to order the DVD and lesson and there should also be a link to the video on YouTube. The individual page for this guide doesn’t have order and download options

**Comment by GenEdProject** [comment]
Fixed so individual pages now have download + add to cart links. Still need to think about the first part. The DVD isn't on sale in teachgenocide.com AFAICT

**Comment by copacetic** [comment]
Again, I need a cost for the DVD, a description, and an image, if we're to add it to our store.

**Comment by Hasmig** [comment]
The DVD and lesson plan should be sold together for $25. Attached is the lesson plan, which we should make downloadable, too.

I was going to grab a describe of this lesson from the press release we posted, but I just noticed that the links to the old press releases are now broken on our website.  That must have happened recently. I'm worried, because I may not have the original press release documents... 

[RI Resource Guide.pdf](https://github.com/GenEdProject/TeachGenocideSite/files/50521/RI.Resource.Guide.pdf)
